any/Unit-Conversion

```diff
@@ -0,0 +1,41 @@
+# Unit Conversion
+
+This expression converts one unit of measurement to another. For example, if a layer contains a measurement field in meters, we can use Arcade expressions to convert the value to feet.
+
+## Use cases
+
+Converting units is useful when you have one numeric field that you'd like to represent with another unit of measurement. This is common in the visualization profile as well as the popup and labels profile. This could be used to visualize the depth of the ocean at different contours in meters or in feet. 
+
+## Workflow
+
+Copy and paste the expression found in the expression template below to 
+the Arcade editor in ArcGIS Online, the relevant location in ArcGIS Pro, or
+the relevant location in a custom app.
+
+To configure the script to your layer, fill in the field name with the measurement field that you would like to convert. 
```

Comment:
To keep this generic, also instruct the user to set the conversion factor in their expression.